packages/frint-config/src/index.js

```diff
@@ -1,5 +1,14 @@
-import externals from './externals';
+import externals, {
+  frint,
+  lodash,
+  thirdParties,
+  webpackRxJs
+} from './externals';
 
 export default {
   externals,
+  frint,
```

Comment:
adding a suffix of `Externals` in all exported keys would make sense. would help identifying which are meant to be used as externals.

since the package is called `frint-config`, in future it may also export other things besides just Webpack externals.

the final export here can be like:

```js
export default {
  externals,

  // new additions:
  frintExternals: frint,
  lodashExternals: lodash,
  rxjsExternals: webpackRxJS,
  thirdPartyExternals: thirdParties,
};
```

Reply:
I didn't consider the `frint-config` would export other members in the future.
Having it declared as `const externals = require('frint-config');` in each example it was the only way I think it would be needed.